prov/verbs/src/fi_verbs.c

```diff
@@ -754,6 +754,7 @@ static void verbs_devs_free(void)
 
 static void vrb_set_peer_mem_support(void)
 {
+#ifndef _WIN32
```

Comment:
It would be better to use `#ifdef` to choose from two versions of the same function rather than commenting out the entire function body.

Reply:
These two functions were almost literally the first issues I encountered and I never looked closely at their usage. Now that I look more closely, they are only called from fi_verbs_ini, a.k.a. VERBS_INI. Assuming you are alright with the conditional compilation around vrb_nd_startup and vrb_nd_shutdown, the simpler thing would probably be to just leave both functions fully intact and conditionally exclude the calls to them by surrounding them with a `#ifndef _WIN32` directive.

Reply:
Yes, that is better.